src/structures/MessageEmbed.js

```diff
@@ -488,7 +489,7 @@ class MessageEmbed {
       type: 'rich',
       description: this.description,
       url: this.url,
-      timestamp: this.timestamp && new Date(this.timestamp),
+      timestamp: this.createdAt?.toISOString(),
```

Comment:
this will be undefined instead of null now, probably not a big issue though

Reply:
That's intended, https://discord.com/developers/docs/resources/channel#embed-object marks it as optional, not nullable